doc/_templates/layout.html

```diff
@@ -76,19 +76,27 @@ <h3>{{ _('Navigation') }}</h3>
 {%- endmacro %}
 
 {%- macro script() %}
-    <script type="text/javascript">
-      var DOCUMENTATION_OPTIONS = {
-        URL_ROOT:    '{{ url_root }}',
-        VERSION:     '{{ release|e }}',
-        COLLAPSE_INDEX: false,
-        FILE_SUFFIX: '{{ '' if no_search_suffix else file_suffix }}',
-        HAS_SOURCE:  {{ has_source|lower }},
-        SOURCELINK_SUFFIX: '{{ sourcelink_suffix }}'
-      };
-    </script>
-    {%- for scriptfile in script_files %}
-    <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}"></script>
-    {%- endfor %}
+    {% if sphinx_version >= "1.8.0" %}
```

Comment:
I think comparing versions as strings is a bad idea. For example, if Sphinx 1.10.0 is released, that version would be considered lower than 1.8.0 with such comparison.

Reply:
I took over the proposed fix (https://github.com/rtfd/sphinx_rtd_theme/pull/672). 
I don't know if Sphinx has a hard policy on using one digit versioning, but at least it looks like 1.8.x is planned to be followed by 2.0.0. 

In the matplotlib python code we use constructs like
https://github.com/matplotlib/matplotlib/blob/2603ebcd66bdd0fd323878c7d420836fbabf0bd3/lib/matplotlib/backend_bases.py#L55-L57

But I'm not familiar enough with the macro language that is used in the html document to judge on whether this could be used there as well.

Reply:
LooseVersion() will actually parse the version string to do a proper comparison, so it's not the same as here.